### PR TITLE
Validation: Duplicate alt text and duplicate description error fixes.

### PR DESCRIPTION
**Pull request by brittmsantos**
## Summary

Fixed:

- image-alt-text-duplicated errors (duplicate alt text descriptions within an article)
- duplicate-descriptions errors (description is duplicated in another article/other articles)

This PR fixes 40 issues across 26 files and is part of ongoing Validation cleanup efforts. No other content has been edited or added.

### CODE REVIEW

**Comment by brittmsantos** [comment]
**Global effort to fix validation errors**

@adegeo 

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

File: dotnet-desktop-guide/framework/winforms/advanced/antialiasing-with-lines-and-curves.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Antialiasing with Lines and Curves"
+description: Learn how GDI+ works with the display driver software to apply antialiasing to pixels within lines and curves.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/advanced/antialiasing-with-lines-and-curves.md:3]:
```suggestion
description: Learn how to apply antialiasing to a line and curve in GDI+ for Windows Forms.
```

File: dotnet-desktop-guide/framework/winforms/advanced/control-help-using-tooltips.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Control Help Using ToolTips"
+description: Learn how to use the System.Windows.Forms.ToolTip component to display a brief, specialized Help message for individual controls on Windows Forms.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/advanced/control-help-using-tooltips.md:3]:
```suggestion
description: Learn how to use the ToolTip component to display a brief, specialized Help message for individual controls on Windows Forms.
```

File: dotnet-desktop-guide/framework/winforms/advanced/cropping-and-scaling-images-in-gdi.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Cropping and Scaling Images in GDI+"
+description: Learn how to use the System.Drawing.Graphics.DrawImage method of the System.Drawing.Graphics class to draw position vector images and raster images.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/advanced/cropping-and-scaling-images-in-gdi.md:3]:
```suggestion
description: Learn how to use the DrawImage method of the System.Drawing.Graphics class to draw and position vector images and raster images.
```

File: dotnet-desktop-guide/framework/winforms/advanced/display-of-asian-characters-with-the-imemode-property.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Display of Asian Characters with the ImeMode Property"
+description: Learn how the System.Windows.Forms.Control.ImeMode property is used by forms and controls to force a mode for an IME used for writing Asian language scripts.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/advanced/display-of-asian-characters-with-the-imemode-property.md:3]:
```suggestion
description: Learn how the Control's ImeMode property is used by forms and controls to force a mode for an IME used for writing Asian language scripts.
```

File: dotnet-desktop-guide/framework/winforms/advanced/double-buffered-graphics.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Double Buffered Graphics"
+description: Learn how to use double buffered graphics in the .NET Framework to reduce flickering in programming graphics.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/advanced/double-buffered-graphics.md:3]:
```suggestion
description: Learn how to use double buffered graphics in the Windows Forms to reduce flickering in programming graphics.
```

File: dotnet-desktop-guide/framework/winforms/advanced/ellipses-and-arcs-in-gdi.md [hunk 6]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Ellipses and Arcs in GDI+"
+description: Learn how to draw ellipses and arcs using the System.Drawing.Graphics.DrawEllipse and System.Drawing.Graphics.DrawArc methods.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/advanced/ellipses-and-arcs-in-gdi.md:3]:
```suggestion
description: Learn how to draw ellipses and arcs using the DrawEllipse and DrawArc methods in GDI+ with Windows Forms.
```

File: dotnet-desktop-guide/framework/winforms/advanced/global-and-local-transformations.md [hunk 7]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Global and Local Transformations"
+description: Learn how global transformations apply to every item drawn by a System.Drawing.Graphics object and local transformations apply to a specific item.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/winforms/advanced/global-and-local-transformations.md:3]:
```suggestion
description: Learn how global transformations apply to every item drawn by a Graphics object and local transformations apply to a specific item.
```

**Review by adegeo — APPROVED** [review]

**Pull request merged**